src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -553,50 +522,15 @@ internal static void MoveToStartElement(XmlReader reader)
             }
         }
 
-        internal static void TraceFeedReadBegin()
-        {
-        }
-
-        internal static void TraceFeedReadEnd()
-        {
-        }
-
-        internal static void TraceFeedWriteBegin()
-        {
-        }
-
-        internal static void TraceFeedWriteEnd()
-        {
-        }
-
-        internal static void TraceItemReadBegin()
-        {
-        }
-
-        internal static void TraceItemReadEnd()
-        {
-        }
-
-        internal static void TraceItemWriteBegin()
-        {
-        }
-
-        internal static void TraceItemWriteEnd()
-        {
-        }
-
-        internal static void TraceSyndicationElementIgnoredOnRead(XmlReader reader)
-        {
-        }
-
         protected abstract SyndicationFeed CreateFeedInstance();
 
         private static T GetNonNullValue<T>(T value, string errorMsg)
         {
             if (value == null)
             {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new InvalidOperationException(SR.Format(errorMsg)));
+                throw new InvalidOperationException(SR.Format(errorMsg));
             }
```

Comment:
SR.Format only makes sense when you have parameters. It basically does the same thing as String.Format. Without any additional parameters, this is a no-op.